index.html

```diff
@@ -1292,10 +1292,11 @@
 
 ### Ancillary uses
 
-[=Sites=] sometimes use data in ways that aren't needed for the user's immediate
+[=Sites=] sometimes use data in ways that aren't needed for the user's primary
 goals. For example, they might bill advertisers, measure site performance, or
-tell developers about bugs. These uses are known as <dfn data-lt="ancillary
-use">ancillary uses</dfn>.
+tell developers about bugs. These uses, which provide indirect support for,
+but are not inherent in, the user's primary goals, are known as
+<dfn data-lt="ancillary use">ancillary uses</dfn>.
```

Comment:
(Apologies for the aborted comment, operator error.)

I agree with @pes10k, but might go further:

```suggestion
tell developers about bugs.

An <dfn>ancillary use</dfn> is any case where [=data processing=]
provides direct benefit to [=actors=]
other than the [=person=] who is the subject of that [=data=].
Ancillary uses of [=personal data=] might benefit to [=people=],
but any benefit is indirect.
For example, being able to charge advertisers provides benefit in that
the site can sustain its business and remain available for future interactions,
but this benefit is primarily realized by business owners.
```

Reply:
Conducting a sales transaction on a website benefits the seller as well as benefiting me. It therefore qualifies as an "ancillary use" of a shopping website under your definition, but I don't think that's what you mean.

Reply:
I see your point, but would suggest that the addition of the word "only" addresses that concern.

Reply:
Ok, well if you're going with this alternate proposal, there are a couple other problems:
1. The paragraph does not connect with the previous paragraph, which is something I tried to ensure with my wording.
2. "Ancillary uses of [=personal data=] might benefit to [=people=]," is ungrammatical and also unclear. What people?

Reply:
It still reads very uncomfortably. Would you consider `s/[=people=]/that [=person=]/`?

And also `s/by business owners/by the site's owners/`.

Because you're mixing up the general and the specific throughout this paragraph and it's awkward.

Reply:
You know, this might be easier with a new pull request, given how far down the rabbit hole we seem to be.  I've opened https://github.com/w3ctag/privacy-principles/pull/459 where you can make suggestions more directly.